generators/generator-bot-enterprise-assistant/README.md

```diff
@@ -1,68 +1,50 @@
+A bot with Azure Language Understanding (LUIS) and common trigger phrases used to direct the conversation flow to help customers accomplish common business tasks. [Learn more](https://aka.ms/EnterpriseAssistant)
 
-Includes support for:
+### Recommended use
 
-- Greeting new and returning users
-- Asking for help
-- Cancelling a dialog
-- Submitting feedback about the bot
-- Error handling in conversations
-- Repeat the previous question
-- Chit chat with QnA Maker ([professional personality](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/how-to/chit-chat-knowledge-base?tabs=v1))
-- Disambiguation of NLP results
+- Create a sophisticated bot
+- Customize and extend sophisticated example dialogs, bot logic, language understanding and bot responses
+
+### Included capabilities
 
-Included skills:
 - [Enterprise Calendar Bot](https://aka.ms/EnterpriseCalendarBot)
 - [Enterprise People Bot](https://aka.ms/EnterprisePeopleBot)
+- Greeting new and returning users
+- Bot Framework Orchestrator to direct the conversation flow
+- Chit chat with QnA Maker ([professional personality](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/how-to/chit-chat-knowledge-base?tabs=v1))
+- Asking for help
+- Disambiguation of user input (Intents)
+- Cancelling a dialog
+- Error handling
+- Repeat previous question
+- Getting customer feedback
 
-## What this template is for
-
-Use this template if you want to...
-
-- Create a conversational experience for common enterprise scenarios (i.e. calendar management and user directory lookup)
-- See examples of more complex conversational flows, and more advanced language understanding and generation
-- See an example of a root bot and skills handling complex user interactions including interruptions and Adaptive Cards
-
-## Packages
-
-Your bot can use the [Azure Bot Framework component model](https://aka.ms/ComponentTemplateDocumentation) to extend the base functionality. From Composer, use the Package Manager to discover additional packages you can add to your bot.
-
-This bot starts with the following packages:
-
-- [Help and Cancel intent handling](https://www.nuget.org/packages/Microsoft.Bot.Components.HelpAndCancel/)
-- [Orchestrator](https://www.nuget.org/packages/Microsoft.Bot.Builder.AI.Orchestrator/)
-
-## Supported Languages
+## Included packages
 
-- English (en-US)
+- [Help and Cancel Intent Handling](https://www.nuget.org/packages/Microsoft.Bot.Components.HelpAndCancel/)
+- [Bot Framework Orchestrator](https://www.nuget.org/packages/Microsoft.Bot.Builder.AI.Orchestrator/)
 
-## Azure Resource Deployment
+The Enterprise Assistant Bot uses packages to extend its capabilities. [Learn more](https://aka.ms/ComponentTemplateDocumentation).
 
```

Comment:
Perhaps special for this particular template should we also include a new section called "Connected skills" or something like that to indicate that this particular bot was extended not just with packages but also with skills as well?

Reply:
we include the skills as the first two items in the included capabilities section.  I'm ok shipping as is.